new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -130,6 +130,11 @@
     try {
       Configuration.setDefaultApiClient(ClientBuilder.defaultClient());
       apiClient = Configuration.getDefaultApiClient();
+      // Set connection and read timeout to 0 to forces the internal HTTP client 
+      // to keep a long running connection with the server to fix SSL connection closed issue
+      apiClient.setConnectTimeout(0);
+      apiClient.setReadTimeout(0);
+      apiClient.setWriteTimeout(0);
```

Comment:
instead of 0 can you set it to some high value so that it times after some time?

Reply:
fixed

Reply:
sorry I have to change the timeout back to 0. because no timeout behaves better than any other number, 10 second(default) 5, 10, 15 or 20 minutes